src/main/java/com/novell/ldapchai/impl/openldap/entry/OpenLDAPPasswordPolicy.java

```diff
@@ -275,7 +279,7 @@ public OpenLDAPPasswordPolicy(final String entryDN, final ChaiProvider chaiProvi
             return (Map) properties;
         }
         catch (IOException e) {
-            // ignore
+            LOGGER.trace("Error opening checkpassword.conf", e);
```

Comment:
'checkpassword.conf' doesn't seem to correspond to any other chai code.  Is this the correct log output?